superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -286,12 +298,17 @@ def convert_to_oe_name(cls, dep, is_native=False):
             dep = dep[:-len('_dev')] + '-rosdev'
         elif dep in ('ros1', 'ros2'):
             dep += '--distro-renamed'
-        return cls.convert_dep_except_oe_vars(dep) \
-            + cls.get_native_suffix(is_native)
+        return cls.modify_name_if_native(
+            cls.convert_dep_except_oe_vars(dep),
+            is_native)
 
     @classmethod
     def generate_multiline_variable(cls, var, container, sort=True, key=None):
         if sort:
+            """
+            TODO: Have default <key> drop trailing '}' so that
```

Comment:
@herb-kuta-lge Please add your GitHub tag

```
TODO(herb-kuta-lge):
```

so that it is clear who to ask if someone wants to resolve the TODO.